.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -11,10 +11,14 @@ assignees: ''
 - [ ] Update changelog if necessary, and merge a PR for it to main
   - [ ] Notify maintainers about updating changelog for in-flight PRs 
 - [ ] Optional: for hotfixes, create a release branch and cherry-pick, e.g. `release-<version>-hotfix`
-- [ ] Optional: create a PR into main from the hotfix branch to see the CI checks pass
+  - [ ] Optional: for hotfixes, create a PR into main from the hotfix branch to see the CI checks pass
 - [ ] On the appropriate branch/commit, create a calver tag for the release candidate, format example: `24.4.0-RC2`
-- [ ] Sign-off with team; confirm tag is correct in #besu-release in Discord
+  - [ ] git tag 24.7.1-RC1
+  - [ ] git push upstream 24.7.1-RC1
+- [ ] Sign-off with team; announce the tag in #besu-release in Discord
+  - [ ] Targeting this tag for the burn-in: 24.7.1-RC1
 - [ ] Consensys staff start burn-in using the proposed release <version-RCX> tag
+- [ ] ... Wait for 3 days
```

Comment:
The number of days could be variable. Not sure we need this line at all tbh...seems implied by "sign off burn-in" to me?